src/main/java/org/terasology/inGameHelp/components/HasBeenHelpedComponent.java

```diff
@@ -17,5 +17,8 @@
 
 import org.terasology.entitySystem.Component;
 
+/**
+ * Component for when the user has been helped
```

Comment:
...and what does this imply? 😛 This particular comment needs to be slightly more specific about what the component does.